### PR TITLE
Fixing pmd points to github file

### PR DESCRIPTION
**Pull request by igieon**
- scm url fix

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -86,9 +86,9 @@
   </prerequisites>
 
   <scm>
-    <connection>scm:svn:http://svn.codehaus.org/mojo/trunk/mojo/mojo-parent</connection>
-    <developerConnection>scm:svn:https://svn.codehaus.org/mojo/trunk/mojo/mojo-parent</developerConnection>
-    <url>http://fisheye.codehaus.org/browse/mojo/trunk/mojo/mojo-parent</url>
+    <connection>scm:svn:https://github.com/mojohaus/mojo-parent</connection>
+    <developerConnection>scm:svn:https://github.com/mojohaus/mojo-parent</developerConnection>
```
Comment (Batmat) [inline pom.xml]: Thanks, but didn't you forget to also update the protocol part? (from svn to Git).
Cheers

**Comment by mfriedenhagen** [comment]
At least the `scm` part seems to be already fixed in 3c045545e51015a73d9dcc13fef492a598290a89

**Comment by mfriedenhagen** [comment]
Partly implemented concerning SCM and the rest duplicates #8.

**Comment by mfriedenhagen** [comment]
I do not think referencing a hash directly is a good way, better attach this as additional artifact.

**Pull request closed without merging**